CMakeLists.txt

```diff
@@ -207,7 +208,6 @@ IF(NOT CYCLUS_DOC_ONLY)
       link_directories(${HDF5_LIBRARY_DIRS})
     endif()
     set(LIBS ${LIBS} ${HDF5_C_LIBRARIES} ${HDF5_C_HL_LIBRARIES})
-    MESSAGE("--    HDF5 Root: ${HDF5_ROOT}")
```

Comment:
Do we want to delete this message? (and the BOOST one?)

Reply:
I added them back in.